core/src/main/java/org/mapstruct/IterableMapping.java

```diff
@@ -21,6 +21,35 @@
  * <p>Note: either  @IterableMapping#dateFormat, @IterableMapping#resultType or @IterableMapping#qualifiedBy
  * must be specified</p>
  *
+ * <p>
+ * <strong>Example:</strong> Convert List&lt;Float&gt; to List&lt;String&gt;
+ * </p>
+ * <pre>
+ * <code>
+ * &#64;Mapper
+ * public interface FloatToStringMapper {
+ *
+ *     &#64;IterableMapping( numberFormat = "##.00" )
+ *     List&lt;String&gt; sourceToTarget(List&lt;Float&gt; source);
+ * }
+ *
+ * // generates
```

Comment:
This is more a general remark:  include the generated code in its own <code> tag